memory-game.js

```diff
@@ -0,0 +1,31 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const grid = document.getElementById("grid-container");
+  function createCard(imgFrontSrc, imgBackSrc) {
+    const cardContainer = document.createElement("div");
+    cardContainer.classList.add("card-container");
+
+    const card = document.createElement("div");
+    card.classList.add("card");
+
+    const imgFront = document.createElement("img");
+    const imgBack = document.createElement("img");
+
+    imgBack.src = imgBackSrc;
+    imgFront.src = imgFrontSrc;
```

Comment:
Excellent code. You're code is easy to read and the variable names are concise and accurate. I very much like how your grouped together lines of code logically belonging together

Reply:
Thank you for your feedback, Bjorn!